portfolio/src/main/webapp/script.js

```diff
@@ -1,4 +1,55 @@
-
 function scrollUp() {
-    window.scrollTo({top:0, behavior: 'smooth'});
-}
\ No newline at end of file
+  window.scrollTo({top: 0, behavior: 'smooth'});
+}
+
+async function printComments() {
+  let commentLimit = document.getElementById('comment-limit').value;
+
+  fetch('/comments?comment-limit=' + commentLimit)
+      .then(response => response.json())
+      .then((data) => {
+        let allComments = document.getElementById('allComments');
+
+        while (allComments.lastChild) {
+          allComments.removeChild(allComments.lastChild)
+        }
+
+        data.forEach((comment) => {
+          createCommentElement(comment);
+        });
```

Comment:
Not sure you need all these brackets... try this
```suggestion
        data.forEach(comment => createCommentElement(comment));
```